src/jit/emitxarch.cpp

```diff
@@ -2469,7 +2472,7 @@ void emitter::emitIns(instruction ins)
                            ins == INS_r_stosb || ins == INS_r_stosd || ins == INS_r_stosp || ins == INS_ret ||
                            ins == INS_sahf || ins == INS_stosb || ins == INS_stosd || ins == INS_stosp
 #ifndef LEGACY_BACKEND
-                           || ins == INS_vzeroupper
+                           || ins == INS_vzeroupper || ins == INS_sfence
```

Comment:
Ditto - this could presumably be as simple as something like: "These are the only instructions that take zero operands"

Reply:
fixed